packages/plugin-workspaces-tools/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@berry/plugin-workspaces-tools",
+  "private": true,
+  "main": "./sources/index.ts",
+  "dependencies": {
+    "@berry/core": "workspace:*",
+    "chalk": "^2.4.1",
+    "clipanion": "^0.14.1",
+    "p-limit": "^2.1.0"
```

Comment:
`p-limit` version 2.2.0 introduced typescript typings, if we switch to that version we can remove the `@ts-ignore` on the import

Reply:
Ahh nice, will do. Thanks!